extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1175,15 +1216,23 @@ export class AmpA4A extends AMP.BaseElement {
       'CLICK_Y': () => {
         return e.pageY;
       },
+      'POSTMESSAGE': (origin, varName) => {
```

Comment:
Its not really "generic" to any postmessage given the above code restricts populating the postmessage data by messages coming from an iframe within an amp-signal-collection-frame element.  I could rename but was just trying to match the i2i #3665 .  Thoughts?